### PR TITLE
XS✔ ◾ Archive rule + Add it to a category

### PR DESCRIPTION
**Pull request by JackDevAU**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

Email `Rules - categories with missing rules`

> 2. What was changed?

Archive rule + Add it to a category

> 3. Did you do pair or mob programming (list names)?

Brady

### CODE REVIEW

File: rules/streamline-development-with-npm-and-task-runners/rule.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 seoDescription: Streamline your .NET development process with NPM and task runners, eliminating tedious tasks and improving collaboration.
 type: rule
-archivedreason:
+archivedreason: "Moved to [Do you know the best package manager for React?](rules/the-best-package-manager-for-react/)"
```
Comment (tiagov8) [inline rules/streamline-development-with-npm-and-task-runners/rule.md:4]: Hey @JackDevAU 

I think you don't need "/rules" in the link here...

**Review by tiagov8 — CHANGES_REQUESTED** [review]

File: rules/streamline-development-with-npm-and-task-runners/rule.md (hunk 1 above)
Comment (JackDevAU) [reply rules/streamline-development-with-npm-and-task-runners/rule.md:4]: I believe for archived rules it needs the `/rules` but for everything else its just a /rule-uri

Comment (JackDevAU) [reply rules/streamline-development-with-npm-and-task-runners/rule.md:4]: So after reviewing it - we actually don't need `/rules` woot

Comment (JackDevAU) [inline rules/streamline-development-with-npm-and-task-runners/rule.md:4]:
```suggestion
archivedreason: "Moved to [Do you know the best package manager for React?](/the-best-package-manager-for-react/)"
```

**Review by bradystroud — APPROVED** [review]

**Pull request merged**